docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -0,0 +1,220 @@
+[[modules-remote-clusters]]
+== Remote clusters
+
+The _remote clusters_ module allows establishing uni-directional connections to
+a remote cluster.
+
+Remote cluster connections work by configuring a remote cluster and connecting
+only to a limited number of nodes in the remote cluster. Each remote cluster is
+referenced by a name and a list of seed nodes.  When a remote cluster is
+registered, its cluster state is retrieved from one of the seed nodes so that up
+to 3 _gateway nodes_ are selected to be connected to as part of remote cluster
+requests.  Remote cluster connections consist of uni-directional connections
+from the coordinating node to the previously selected remote nodes only. It is
+possible to tag which nodes should be selected through node attributes (see
+<<remote-cluster-settings>>).
+
+Each node in a cluster that has remote clusters configured connects to one or
+more _gateway nodes_ and uses them to federate requests to the remote cluster.
+
+ifndef::include-xpack[]
+This functionality is used in
+<<modules-cross-cluster-search,cross-cluster search>>, and possibly other
+features.
+endif::[]
+ifdef::include-xpack[]
+This functionality is used in cross-cluster replication,
+<<modules-cross-cluster-search,cross-cluster search>>, and posssibly other
+features.
+endif::[]
+
+[float]
+[[configuring-remote-clusters]]
+=== Configuring Remote Clusters
+
+Remote clusters can be specified globally using
+<<cluster-update-settings,cluster settings>> (which can be updated dynamically),
+or local to individual nodes using the `elasticsearch.yml` file.
+
+If a remote cluster is configured via `elasticsearch.yml` only the nodes with
+that configuration will be able to connect to the remote cluster. In other
+words, functionality that relies on remote cluster requests will have to be
```

Comment:
This seems like a very particular feature. Almost like we'd be better off giving the advice to set the remote cluster up in every `elasticsearch.yml`.